docs/azure_jumpstart_hcibox/deployment_azd/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+type: docs
+linkTitle: "Deploy (azd)"
```

Comment:
I think the title should not use an acronym. I suggest "Azure Developer CLI deployment"